.travis.yml

```diff
@@ -16,7 +16,9 @@ rvm:
 
 matrix:
   allow_failures:
+    - rvm: 1.9.3
```

Comment:
We should remove this, no need to support anything < ruby 2.0 and maybe even 2.2 I would say.